Modules/Devices/test/Devices.Application.Tests/Tests/Identities/Commands/SendDeletionProcessGracePeriodReminders/HandlerTests.cs

```diff
@@ -58,11 +58,11 @@ public async Task Sends_first_reminder()
 
         // Assert
         A.CallTo(() => mockPushNotificationSender.SendNotification(identity.Address, A<DeletionProcessGracePeriodReminderPushNotification>._, A<CancellationToken>._))
-            .MustHaveHappenedOnceExactly();
+            .MustNotHaveHappened();
```

Comment:
As said above, I don't think these should be blocked.

Reply:
Changes made.